src/CMakeLists.txt

```diff
@@ -297,28 +297,19 @@ endif ()
 # DBCSR LIBRARY's HIP BACKEND
 
 if (USE_HIP)
-
-  if (USE_OPENMP)
-    set(HIP_HIPCC_FLAGS "${HIP_HIPCC_FLAGS} ${OpenMP_CXX_FLAGS}")
-  endif ()
-
   # add libsmm_acc which has its own CMakeLists.txt due to code generation
   add_subdirectory(acc/libsmm_acc)
   target_link_libraries(dbcsr PUBLIC libsmm_acc)
-  target_link_libraries(dbcsr PUBLIC ${ROCM_HIPRTC_LIB})
+  target_link_libraries(dbcsr PUBLIC hip::amdhip64)
 
   # Complete list of GPU-support sources
   set(DBCSR_ACC_SRCS ${DBCSR_HIP_SRCS})
 
   # Compile the rest of the HIP source files into a static library
-  set_source_files_properties(${DBCSR_ACC_SRCS}
-                              PROPERTIES HIP_SOURCE_PROPERTY_FORMAT 1)
-  hip_add_library(acc STATIC ${DBCSR_ACC_SRCS})
-  target_include_directories(acc PRIVATE ${HIP_PATH}/../include)
+  add_library(acc SHARED ${DBCSR_ACC_SRCS})
```

Comment:
Do we have to build a full SHARED lib her now?
How does that work with the rest of DBCSR being built as a static library (by default)?
Could we use an `OBJECT` here now (getting only one lib in the end)?

Reply:
Let me try, I took the old comments above this as if it was intended to build a shared lib, but it wasn't supported by hip_add_library at that point in time.